src/components/header/settingsButton/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import {styles} from "./styles";
+import {useNavigation} from "@react-navigation/native";
+import {Routes} from "../../../constants";
+import IconButton from "../../IconButton";
+
+const SettingsButton = () => {
+    const navigation = useNavigation();
+
+    const handleButtonPress = () => {
+        // FIXME: ###CF### Use correct type
+        // @ts-ignore
```

Comment:
Should this be fixed within this PR?

Reply:
Yeah, but I still have no any idea about correct type for it :)